settings/common.py

```diff
@@ -156,6 +156,8 @@
 
 SITE_ID = 1
 
+REST_USE_JWT = True
```

Comment:
Why do we need to add this?

Reply:
This changed is need for `rest_auth` to verify user through `JWT`.